src/momento/aio/_scs_data_client.py

```diff
@@ -42,13 +41,11 @@ def __init__(self, configuration: Configuration, credential_provider: Credential
         endpoint = credential_provider.get_cache_endpoint()
         self._logger = logs.logger
         self._logger.debug("Simple cache data client instantiated with endpoint: %s", endpoint)
-        default_deadline: timedelta = (
-            configuration.get_transport_strategy().get_grpc_configuration().get_deadline() or _DEFAULT_DEADLINE_SECONDS
-        )
+        default_deadline: timedelta = get_default_client_deadline(configuration)
```

Comment:
I was thinking this could be a method on TransportStrategy, no function wrapper. It can also hide that we're using the GRPC deadline as a timeout.

TransportStrategy defines an abstract `get_client_timeout` and StaticTransportStrategy implements it. The default lives in StaticTransportStrategy.

I think we should avoid the GRPC term "deadline" when the transport strategy is hiding GRPC. We even [pass the "deadline" as a "timeout" to the GRPC stub](https://github.com/momentohq/client-sdk-python/blob/51ecff677cee47ab7d0012838399f3bcaf5ec454/src/momento/aio/_scs_data_client.py#L65).

```
default_timeout: configuration.get_transport_strategy.get_client_timeout()
```